CPAC/pipeline/engine.py

```diff
@@ -1926,7 +1932,17 @@ def ingress_raw_anat_data(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id
             dl_dir=cfg.pipeline_setup["working_directory"]["path"],
             img_type="anat",
         )
-        rpool.set_data("T1w", anat_flow, "outputspec.anat", {}, "", "anat_ingress")
+        reorient = pe.Node(
+            interface=afni.Resample(),
+            name=f"reorient_T1w_{part_id}_{ses_id}",
+        )
+
+        reorient.inputs.orientation = desired_orientation
+        reorient.inputs.outputtype = "NIFTI_GZ"
+
+        wf.connect(anat_flow, "outputspec.anat", reorient, "in_file")
+
+        rpool.set_data("T1w", reorient, "out_file", {}, "", "anat_ingress")
```

Comment:
Instead of doing this several times, we could add a method to [the `ResourcePool` class](https://github.com/FCP-INDI/C-PAC/blob/1a5ba5c40d119ee18248a51cfa17a3fb59afed98/CPAC/pipeline/engine.py#L73-L1417) like

```Python
    def reorient_node(
        self,
        resource_key: str,
        node_name: Optional[str] = None,
        id: Optional[str] = None,
        set_data: bool = True
    ) -> pe.Node:
        """Return a Node to reorient a resource."""
        if set_data and not node_name:
            msg = "`reorient_node` requires a `node_name` if `set_data=True`"
            raise ValueError(msg)
        _name = f"reorient_{resource_key.replace('-', '_')}"  # I think Nipype wants underscored instead of dashes in Node names? If it doesn't matter, then this doesn't need the "replace"
        if id:
           _name = f"{name}_{id}"
        reorient = pe.Node(
            interface=afni.Resample(),
            name=_name,
        )
        reorient.inputs.orientation = self.cfg.pipeline_setup["desired_orientation"]
        reorient.inputs.outputtype = "NIFTI_GZ"
        if set_data:
            self.set_data(resource_key, reorient, "out_file", {}, "", node_name)
        return reorient
```

that returns a Node that can be connected, then this whole section can just be
```suggestion
        reorient = rpool.reorient_node("T1w", "anat_ingress", f"{part_id}_{ses_id}")
        wf.connect(anat_flow, "outputspec.anat", reorient, "in_file")
```